### PR TITLE
Add Serein Scan

### LINKED ISSUE #2301: Source request for Serein Scan
### Source name

Serein Scan

### Source link

https://sereinscan.com/

### Source language

Turkish

### Other details

_No response_

### Acknowledgements

- [X] I have checked that the extension does not already exist by searching the [extension listing](https://keiyoushi.github.io/extensions/) and verified it does not appear there.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a meaningful title with the source name.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by Norsze**
Closes  #2301

Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [ ] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [x] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

File: src/tr/sereinscan/build.gradle [hunk 1]
```diff
@@ -0,0 +1,9 @@
+ext {
+    extName = 'Serein Scan'
+    extClass = '.SereinScan'
+    themePkg = 'mangathemesia'
+    baseUrl = 'https://sereinscan.com/'
```
Comment (vetleledaal) [inline src/tr/sereinscan/build.gradle]:
```suggestion
    baseUrl = 'https://sereinscan.com'
```

File: src/tr/sereinscan/src/eu/kanade/tachiyomi/extension/tr/sereinscan/SereinScan.kt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package eu.kanade.tachiyomi.extension.tr.sereinscan
+
+import eu.kanade.tachiyomi.multisrc.mangathemesia.MangaThemesia
+
+class SereinScan : MangaThemesia(
+    "Serein Scan",
+    "https://sereinscan.com/",
```
Comment (vetleledaal) [inline src/tr/sereinscan/src/eu/kanade/tachiyomi/extension/tr/sereinscan/SereinScan.kt]:
```suggestion
    "https://sereinscan.com",
```

File: src/tr/sereinscan/src/eu/kanade/tachiyomi/extension/tr/sereinscan/SereinScan.kt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+package eu.kanade.tachiyomi.extension.tr.sereinscan
+
+import eu.kanade.tachiyomi.multisrc.mangathemesia.MangaThemesia
+
+class SereinScan : MangaThemesia(
+    "Serein Scan",
+    "https://sereinscan.com",
+    "tr",
+)
```
Comment (AwkwardPeak7) [inline src/tr/sereinscan/src/eu/kanade/tachiyomi/extension/tr/sereinscan/SereinScan.kt:12]: site uses turkish dates, update the dateFormat too.

Comment (Norsze) [reply src/tr/sereinscan/src/eu/kanade/tachiyomi/extension/tr/sereinscan/SereinScan.kt:12]: like this?

Comment (AwkwardPeak7) [reply src/tr/sereinscan/src/eu/kanade/tachiyomi/extension/tr/sereinscan/SereinScan.kt:12]: yea, does it work?

Comment (Norsze) [reply src/tr/sereinscan/src/eu/kanade/tachiyomi/extension/tr/sereinscan/SereinScan.kt:12]: seems to, it shows the date that I see on the website for the entries I looked at

**Review by AwkwardPeak7 — APPROVED** [review]

**Pull request merged**